service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2819,7 +2819,9 @@ components:
           items:
             type: string
         limit:
-          $ref: "#/components/schemas/Limit"
+          type: integer
```

Comment:
This was probably because done unknowingly because there's a `Limit` parameter which does have a maximum of 50 which is different from the component version. I didn't look enough to see what the best change would be but probably one should get renamed at least.

Reply:
From our discussion - found the other param_limit. Refactored it to re-use the same schema object with same default of 50

Reply:
Update - reusing the schema object does not work (default and required fields are ignored). renamed the param instead